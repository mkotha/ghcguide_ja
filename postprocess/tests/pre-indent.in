<html>
<body>
<pre>
  module Main where

  main :: IO ()
  main = return ()
</pre>
</body>
</html>
