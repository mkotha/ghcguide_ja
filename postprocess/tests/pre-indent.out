
<html lang="en" xml:lang="en" xmlns="http://www.w3.org/1999/xhtml">
  <body>

<pre>
module Main where

main :: IO ()
main = return ()
</pre>

  </body>
</html>
